lib/elixir/lib/regex.ex

```diff
@@ -159,6 +159,7 @@ defmodule Regex do
       "Unicode-spaces"
 
   """
+  alias DialyxirVendored.Warnings.NoReturn
```

Comment:
Hrm. This was added by VS code when i added NoReturn to the `@spec` for to_embed/2. I suspect that isnt quite right.

Reply:
Removed in the latest version.